### PR TITLE
JAMES-2870 update CHANGELOG.md about zookeeper deprecation and OSGi r…

### PR DESCRIPTION
**Pull request by mbaechler**
…emoval

### CODE REVIEW

**Comment by mbaechler** [comment]
We don't need the build result for that, I stopped the job.

**Comment by chibenwa** [comment]
test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Migration to ElasticSearch 6.3
 - Blob export to LinShare
 
+### Deprecated
+- Zookeeper component. This will be removed after 3.4 release. If you have development skills, and are willing to maintain this component, please reach us.
+
+### Removed
+- Karaf OSGi implementation. It was not compiled nor tested for a year. We didn't got any feedback about that and the team doesn't have the resources nor the skills to maintain it any more.
```
Comment (Arsnael) [inline CHANGELOG.md]: s/didn't got/didn't get

Comment (rouazana) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Migration to ElasticSearch 6.3
 - Blob export to LinShare
 
+### Deprecated
+- Zookeeper component. This will be removed after 3.4 release. If you have development skills, and are willing to maintain this component, please reach us.
+
+### Removed
+- Karaf OSGi implementation. It was not compiled nor tested for a year. We didn't get any feedback about that and the team doesn't have the resources nor the skills to maintain it any more.
```
Comment (blackheaven) [inline CHANGELOG.md]: I'm not easy with 'the team', we are supposed to be a community

Comment (rouazana) [reply CHANGELOG.md]: fixed

**Comment by rouazana** [comment]
merged

**Pull request closed without merging**